packages/cheqd/src/anoncreds/utils/identifiers.ts

```diff
@@ -9,18 +9,23 @@ const IDENTIFIER = `((?:${ID_CHAR}*:)*(${ID_CHAR}+))`
 const PATH = `(/[^#?]*)?`
 const QUERY = `([?][^#]*)?`
 const VERSION_ID = `(.*?)`
+const FRAGMENT = `([#].*)?`
```

Comment:
Patches that I found while working on this PR. I tested issuance of OpenID4VC with cheqd, and it was failing because of this.

And instead of making the right choice to create a separate PR i added it all to this one 🙃